pykob/preferencesWindow.py

```diff
@@ -238,7 +238,7 @@ def __init__(self, callback=None, quitWhenDismissed=False):
                             variable=self.codeType,
                             value=codeTypeRadioButton + 1).grid(row=6, column=1 + codeTypeRadioButton, sticky=tk.W)
             # If current config matches this radio button, update the selected value
-            if config.code_type.to_string() == self.CODE_TYPE_SETTINGS[codeTypeRadioButton]:
+            if config.code_type.to_string().upper() == self.CODE_TYPE_SETTINGS[codeTypeRadioButton].upper():
```

Comment:
You can use `config.code_type.name` rather than `to_string()`. Likewise, `config.code_type.name.upper()`. It is more clear to read.